### PR TITLE
Add 1E Datadog integration

### PR DESCRIPTION
**Pull request by Sid-1E**
### What does this PR do?

Initial PR for 1E integration tile

### Motivation

Engaged with the Datadog partnership team.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: 1e/README.md
Comment (hestonhoffman) [inline 1e/README.md:11]:
```suggestion
Getting set up with the 1E Datadog integration is as simple as emailing a Datadog API key and region to the 1E Support team. There's no further configuration required.
```

File: 1e/README.md
Comment (hestonhoffman) [inline 1e/README.md:13]:
```suggestion
1. Go to your [Datadog Integrations page][1] and click on the 1E tile. 
1. Click the **Configuration** tab and click **Install Integration** at the bottom.
```

File: 1e/README.md
Comment (hestonhoffman) [inline 1e/README.md:20]:
```suggestion
   - If the domain name is `app.datadoghq.com`, your API region is `US`.
   - If the domain name is `app.datadoghq.eu`, your API region is `EU`.
```

File: 1e/README.md
Comment (hestonhoffman) [inline 1e/README.md:22]:
```suggestion
4. Send an email to 1E support with the Datadog API key and region to activate your 1E Datadog integration.
```

File: 1e/README.md
Comment (hestonhoffman) [inline 1e/README.md:24]:
```suggestion
5. After you receive a reply from the support team, go to the Datadog metrics explorer to see your metrics.
```

File: 1e/README.md
Comment (hestonhoffman) [inline 1e/README.md:30]:
```suggestion
See [metadata.csv][3] for a list of metrics provided by this integration.
```

File: 1e/README.md
Comment (hestonhoffman) [inline 1e/README.md:34]:
```suggestion
The 1E integration does not include any service checks.
```

File: 1e/README.md
Comment (hestonhoffman) [inline 1e/README.md:38]:
```suggestion
The 1E integration does not include any events.
```

**Review by hestonhoffman — APPROVED** [review]

File: 1e/metadata.csv
Comment (ofek) [inline 1e/metadata.csv:2]: The metrics need a prefix that conforms to https://docs.datadoghq.com/developers/guide/what-best-practices-are-recommended-for-naming-metrics-and-tags/#rules-and-best-practices-for-naming-metrics

**Review by ofek — CHANGES_REQUESTED** [review]

File: 1e/README.md
Comment (emarsha94) [inline 1e/README.md:5]: We recommend providing more insight on the value of the integration and the integrations purpose

File: 1e/manifest.json
Comment (emarsha94) [inline 1e/manifest.json:15]:
```suggestion
      "Offering::Integration",
      "Supported OS::Linux",
```

File: 1e/assets/service_checks.json
Comment (emarsha94) [inline 1e/assets/service_checks.json:1]: This file can be removed since it does not include any service checks

File: 1e/README.md
Comment (emarsha94) [inline 1e/README.md:24]: Would be good to link the 1E support email

**Review by emarsha94 — CHANGES_REQUESTED** [review]

**Review by FlorentClarret — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks good to me but left a few comments and suggestions.

Also, I'm updating the validation command your make the CI green.

File: 1e/metadata.csv
Comment (Sid-1E) [reply 1e/metadata.csv]: Added onee as prefix

File: 1e/manifest.json
Comment (Sid-1E) [reply 1e/manifest.json]: one_e is a better option, but we have already added onee as prefix to API service which is publishing data to Dashboard.

**Comment by FlorentClarret** [comment]
Hi! Thanks again for all the modifications. Could you add again the `service_check.json` file, even if empty? We'll update the validation command in a follow up PR to fix this. Except that, looks good to me.

**Comment by Sid-1E** [comment]
> Hi! Thanks again for all the modifications. Could you add again the `service_check.json` file, even if empty? We'll update the validation command in a follow up PR to fix this. Except that, looks good to me.

Hi, I have added the `service_check.json` file. Thanks

**Comment by Sid-1E** [comment]
Hi,
As this PR is closed now, but not merged to master. Can you let me know if there is anything required from our end, or merging will be done by Datadog team.
We need this tile to be part of Datadog integration before Dash.
Thanks

**Comment by FlorentClarret** [comment]
Hi @Sid-1E, for this to be merged, you need to reopen the PR and we will merge it once everything is ready

File: 1e/CHANGELOG.md
Comment (emarsha94) [inline 1e/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2022-10-20
```

File: 1e/README.md
Comment (emarsha94) [inline 1e/README.md:5]:
```suggestion
The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account. You can use these metrics to take advantage of the out-of-the-box dashboard or you can create your own visualizations.
```

File: 1e/README.md
Comment (emarsha94) [inline 1e/README.md:17]:
```suggestion
3. Go to your [Datadog API Keys Management page][2] and create an API key.
```

File: 1e/README.md
Comment (emarsha94) [inline 1e/README.md:19]: Datadog API Region should be changed to Datadog Site as this is how we identify regions
```suggestion
4. Determine your Datadog site by looking at your browser's address bar:
```

File: 1e/README.md
Comment (emarsha94) [inline 1e/README.md:21]: Please replace region with site throughout the steps

File: 1e/README.md
Comment (emarsha94) [reply 1e/README.md:19]: I recommend hyperlinking our datadog site doc: https://docs.datadoghq.com/getting_started/site/

File: 1e/README.md
Comment (emarsha94) [inline 1e/README.md:26]:
```suggestion
6. 1Es team will be in touch once your account has been set up. Once it's been setup, navigate to the Metrics Explorer within Datadog to see metrics begin to flow in.
```

Comment (emarsha94) [reply 1e/README.md:26]: I recommend including a hyperlink to the Metric Explorer page as well

File: 1e/manifest.json
Comment (emarsha94) [inline 1e/manifest.json:12]:
```suggestion
    "title": "1E Integration",
```

File: 1e/manifest.json
Comment (emarsha94) [inline 1e/manifest.json:11]: This should be a brief description on what 1E is and offering

Comment (emarsha94) [inline 1e/manifest.json:11]: This should be a brief description on what 1E is and the offering

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: 1e/README.md
Comment (emarsha94) [inline 1e/README.md:24]: We recommend having the user input their api key somewhere on 1Es site since email isn't a secure way to send api keys.

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: 1e/README.md
Comment (Sid-1E) [reply 1e/README.md]: @emarsha94 Thanks for the suggestion. Can we consider this change in next PR or may be if these changes be released to staging environment for Dash showcase.

File: 1e/manifest.json
Comment (FlorentClarret) [inline 1e/manifest.json:24]:
```suggestion
      "source_type_name": "one-e",
```

for consistency with the `app_id`

Comment (FlorentClarret) [reply 1e/manifest.json:24]: Applying to speed up the process

File: 1e/manifest.json
Comment (FlorentClarret) [inline 1e/manifest.json:11]: nit
```suggestion
    "description": "Send your 1E products metrics to Datadog.",
```

**Review by FlorentClarret — APPROVED** [review]
Looks good to me. 

Waiting on @emarsha94 approval to merge this PR.

File: 1e/README.md
Comment (anawishnoff) [reply 1e/README.md]: Hi @Sid-1E, I'm a PM on the same team as @emarsha94. Unfortunately this is a hard blocker for our team - we can't allow for the transfer of API Keys over email as it poses a large security risk. The pattern we normally suggest is to have users enter their credentials on some kind of integration configuration page on your website. You could then pull those and use them in any subsequent calls to the Datadog API.

File: 1e/README.md
Comment (anawishnoff) [inline 1e/README.md:5]:
```suggestion
The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account. These metrics are visualized on the out-of-the-box dashboard, and can also be used to  create your own visualizations. 
```

File: 1e/README.md
Comment (anawishnoff) [inline 1e/README.md:11]:
```suggestion
Getting set up with the 1E Datadog integration is simple, and requires the following:
```

File: 1e/README.md
Comment (anawishnoff) [inline 1e/README.md:13]:
```suggestion
```
This step doesn't seem to be necessary, because if the user is reading this they'll already be on the tile :)

File: 1e/README.md
Comment (anawishnoff) [inline 1e/README.md:15]:
```suggestion
```
Removing for same reason as above - these instructions appear on the configure tab

File: 1e/README.md
Comment (anawishnoff) [inline 1e/README.md:17]:
```suggestion
3. After clicking "Install Integration" on this tile, go to your [Datadog API Keys Management page][2] and create an API key.
```

File: 1e/README.md
Comment (anawishnoff) [inline 1e/README.md:22]: We have more sites than just US and EU - I'd recommend removing these two sub-bullets and prompting the user to use the documentation linked to determine their site.

File: 1e/manifest.json
Comment (anawishnoff) [inline 1e/manifest.json:11]:
```suggestion
    "description": "Send your 1E product metrics to Datadog.",
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Requesting changes on behalf of the product team

File: 1e/manifest.json
Comment (mgarabed) [reply 1e/manifest.json:24]: This doesn't look like it got changed to "one-e"?

File: 1e/README.md
Comment (eho1307) [inline 1e/README.md:5]: Non-blocking suggestion but it would be good to add one line about what 1E does.

```suggestion
1E is a digital experience solution that automatically surfaces user experience issues with recommended troubleshooting steps, reducing points of friction. The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account. These metrics are visualized on the out-of-the-box dashboard, and can also be used to  create your own visualizations. 
```

File: 1e/README.md
Comment (Sid-1E) [reply 1e/README.md]: We have discussed with our support team, and will ask customers to provide this detail via our secure 1E Support Portal. I have changed the set up steps accordingly.

**Pull request closed without merging**
